MCR-1978 Validate Session IP with Netmask

OPNA2608 opened the pull request:
Validating the session's last IP against the new IP with configurable netmasks
"MCR.Servlet.Session.NetMask.IPv4" and "MCR.Servlet.Session.NetMask.IPv6".

[Link to jira](https://mycore.atlassian.net/browse/MCR-).

OPNA2608 committed:
MCR-1978 Validate Session IP with Netmask

Validating the session's last IP against the new IP with configurable netmasks
"MCR.Servlet.Session.NetMask.IPv4" and "MCR.Servlet.Session.NetMask.IPv6".

kkrebs commented on mycore-base/src/main/java/org/mycore/frontend/servlets/MCRServlet.java:
```diff
@@ -87,6 +91,16 @@
         "MCR.Servlet.BrowserCache.enable", false);
 
+    private static final String SESSION_NETMASK_IPV4_STRING = MCRConfiguration.instance()
+        .getString("MCR.Servlet.Session.NetMask.IPv4", "255.255.255.255");
```
these properties should also be part of mycore.properties and documentated there

OPNA2608 replied:
Done.

kkrebs commented on mycore-base/src/main/java/org/mycore/frontend/servlets/MCRServlet.java:
```diff
@@ -87,6 +91,16 @@
+        .getString("MCR.Servlet.Session.NetMask.IPv4", "255.255.255.255");
+
+    private static final String SESSION_NETMASK_IPV6_STRING = MCRConfiguration.instance()
+        .getString("MCR.Servlet.Session.NetMask.IPv6", "FFFF:FFFF:FFFF:FFFF:FFFF:FFFF:FFFF:FFFF");
```
dito

OPNA2608 replied:
Done².

OPNA2608 committed:
MCR-1978 New Properties in mycore.properties

Added the new Netmask properties with a short explaination
to the mycore-base mycore.properties.
